src/DynamoCoreWpf/Interfaces/IWatchHandler.cs

```diff
@@ -24,16 +24,17 @@ namespace Dynamo.Interfaces
     public interface IWatchHandler
     {
         event Action<string> RequestSelectGeometry;
-        WatchViewModel Process(dynamic value, ProtoCore.RuntimeCore runtimeCore, string tag, bool showRawData, WatchHandlerCallback callback);
+        WatchViewModel Process(dynamic value, IEnumerable<string> preferredDictionaryOrdering, ProtoCore.RuntimeCore runtimeCore, string tag, bool showRawData, WatchHandlerCallback callback);
```

Comment:
Couldn't this simply be a list? This looks likely to result in multiple enumerations.